README.md

```diff
@@ -549,6 +576,7 @@ To upgrade to version 8, the following changes should be made.
 - A `sizes` prop should be added to all usages of Imgix. If `sizes` is new to you (or even if it's not), Eric's [seminal article on `srcset` and `sizes`](https://ericportis.com/posts/2014/srcset-sizes/) is highly recommended.
 - Change all usages of `type='picture'` to `<Picture>` and `type='source'` to `<Source>`
 
+```html
```

Comment:
I believe you actually want to indent these blocks with four spaces, in order for it to render correctly in GitHub:

```
    ```html
    // this...
    <Imgix type='picture'>
      <Imgix type='source' src={src}>
      <Imgix type='source' src={src}>
    </Imgix>
    // becomes...
    <Picture>
      <Source src={src}>
      <Source src={src}>
    </Picture>
    ```
```

Reply:
I think you missed my point--the entire block needs to be indented four spaces.

Reply:
Hopefully that did it